Composer/packages/lib/indexers/src/dialogIndexer.ts

```diff
@@ -223,6 +223,7 @@ function index(files: FileInfo[], botName: string): DialogInfo[] {
           const isRoot = file.relativePath.includes('/') === false; // root dialog should be in root path
           const dialog: DialogInfo = {
             isRoot,
+            isTopic: file.relativePath.startsWith('topics/'),
```

Comment:
I think the answer is yes, but just confirming: We are fine with not indexing the topic if the user decides to muck around with the folder structure right?

Reply:
We should engage the PVA team on that. My assumption is that the folder structure is necessary when publishing back to PVA, but I don't know for sure.